### PR TITLE
Delete monitor: tweak UI for modal pop  

### PR DESCRIPTION
**Pull request by lovedeep92**
**JIRA:**
See ticket here - https://jira.rax.io/browse/MNRVA-582

**Description:**
1) We need to show error and success icon according to the response coming from an api when you delete any monitor.
2) if monitor is deleted then the Alert icon should show in the modal popup
3) If a monitor is deleted then the Success icon should show in a modal popup.

**Testing:**

npm run test:unit

**Screenshots:
Screenshot 2020-10-21 at 12 12 34 AM
**
(if applicable)

### CODE REVIEW

**Review by pras7430 — COMMENTED** [review]
Functionality is not working as expected for Mock as well as in LocalEnv.

1. Select all and then unselect a few of them and then delete them.
2. Count is not working properly.

select all and then unselect few
Screenshot 2020-10-23 at 7 14 20 PM
Screenshot 2020-10-23 at 6 54 00 PM

File: src/app/_features/monitors/components/list/monitorslist.component.ts [hunk 1]
```diff
@@ -185,6 +190,11 @@ export class MonitorslistComponent implements OnInit {
     });  
     this.selectedMonitors = [];
     this.fetchMonitors();
+    this.selectedMonitors = this.monitors.map(x => Object.assign({}, x));
+    if(this.failedMonitors.length > 0) {
```
Comment (FarajiA) [inline src/app/_features/monitors/components/list/monitorslist.component.ts:194]: We should add a test for this logic. If `failedMonitors` has items in its array let's spy on the logService to make sure it was called as intended.

**Review by FarajiA — CHANGES_REQUESTED** [review]
Be sure to grab the updates to the tests I added.

File: src/app/_features/monitors/components/list/monitorslist.component.ts (hunk 1 above)
Comment (lovedeep92) [reply src/app/_features/monitors/components/list/monitorslist.component.ts:194]: okay !!

**Comment by lovedeep92** [comment]
@pras7430 looking into it!

**Comment by lovedeep92** [comment]
@pras7430 fixed!

**Comment by lovedeep92** [comment]
@FarajiA its done. take latest pull and check!

**Review by FarajiA — APPROVED** [review]
nice

**Pull request merged**